frontend/src/insecure/XSSInnerHTML.jsx

```
@@ -0,0 +1,8 @@
+/**
+ * Dangerous innerHTML use.
+ */
+import React from "react";
+
+export default function XSSInnerHTML({ html }){
+  return <div dangerouslySetInnerHTML={{ __html: html }} />;
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)